README.md

```diff
@@ -235,6 +236,12 @@ Open a terminal with Ctrl+Alt+T and run:
 ```bash
 bash <(curl -Ls bit.ly/run-aurora) install_python3
 ```
+# Multiple Aurora Installations on one device #
+
+Aurora installs icons by taking the containers name and adding it too both ".shadow_launcher_app_{container_name}" and "Shadow Launcher {container_name}.desktop". This lets us have control over which icons get removed and reinstalled when rerunning Aurora.
+
+This allows us to easily have multiple Aurora setups on one device, you simply need to define a unique name for the variable container_name by adding "container_name=new_container_name" to your oneliner when running each Aurora oneliner.
```

Comment:
This section should go above and generally describe a bit better the purpose of this change